subsys/net/lib/http/http_server_core.c

```diff
@@ -704,6 +712,44 @@ struct http_resource_detail *get_resource_detail(const char *path,
 	return NULL;
 }
 
+int http_server_find_file(char *fname, size_t fname_size, size_t *file_size, bool *gzipped)
+{
+	struct fs_dirent dirent;
+	size_t len;
+	int ret;
+
+	ret = fs_stat(fname, &dirent);
+	if (ret < 0) {
+		len = strlen(fname);
+		snprintk(fname + len, fname_size - len, ".gz");
+		ret = fs_stat(fname, &dirent);
+		*gzipped = (ret == 0);
+	}
+
+	if (ret == 0) {
+		*file_size = dirent.size;
+		return ret;
+	}
+
+	return -ENOENT;
+}
+
+void http_server_get_content_type_from_extension(char *url, char *content_type,
+						 size_t content_type_size)
+{
+	size_t url_len = strlen(url);
+
+	HTTP_SERVER_CONTENT_TYPE_FOREACH(ct)
+	{
```

Comment:
Please place the starting brace to previous line

Reply:
do you want me to add the `HTTP_SERVER_CONTENT_TYPE_FOREACH` to the `ForEachMacros` list in zephyr's .clang-format file?

Reply:
Is following zephyr style causing some issue with clang-format?

Reply:
clang-format isn't exactly formatting wrong, it just doesn't know about FOREACH macros and seems to think it's a function and therefore moves the opening brace to a newline.